airflow/operators/bash_operator.py

```diff
@@ -70,10 +71,10 @@ class BashOperator(BaseOperator):
     @apply_defaults
     def __init__(
             self,
-            bash_command,
-            env=None,
-            output_encoding='utf-8',
-            *args, **kwargs):
+            bash_command: str,
+            env: Dict[str, str] = None,
```

Comment:
If the default is `None`, this should be `Optional[Dict[str, str]]`.

Reply:
This is not required. mypy is smart and it can guess.
Reference: https://mypy.readthedocs.io/en/stable/command_line.html#none-and-optional-handling